client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -109,16 +109,16 @@ class PurchasesListing extends Component {
 	getTagline( purchase ) {
 		const { currentPlanSlug, translate } = this.props;
 
-		if ( isJetpackBackup( purchase ) ) {
+		if ( isJetpackProduct( purchase ) ) {
 			return getJetpackProductTagline( purchase );
 		}
 
-		const jetpackBackupPurchase = this.getJetpackBackupPurchase();
+		const jetpackBackupPurchase = this.getJetpackProductPurchases();
```

Comment:
Not necessarily a backup purchase at this point, so perhaps this constant should be renamed to avoid confusion?

Reply:
Just pushed a fix. Thanks for catching that.